views/js/helper/status.js

```diff
@@ -67,21 +67,21 @@ define(['lodash', 'i18n'], function(_, __){
             code : _canceled,
             label : __('Canceled'),
             can : {
-                authorize : __('is canceled'),
-                pause : __('is canceled'),
-                terminate : __('is canceled'),
+                authorize : __('canceled'),
+                pause : __('canceled'),
+                terminate : __('canceled'),
```

Comment:
It seems you are using these labels as a key to get the translated message. If so you should not translate them, otherwise you will go in trouble if those labels are translated to something else...